apisix/cli/ngx_tpl.lua

```diff
@@ -631,12 +631,15 @@ http {
         {% if enable_http2 then %}
         http2 on;
         {% end %}
+        {% if enable_http3_global then %}
```

Comment:
Why is there a `global` suffix?

Reply:
Does that mean http3 is enabled globally?

Reply:
> Does that mean http3 is enabled globally?

yes, I use `enable_http3` to determine whether to add the `quic` parameter in [listen directive](https://nginx.org/en/docs/http/ngx_http_core_module.html)
> The quic parameter (1.25.0) configures the port to accept [QUIC](https://nginx.org/en/docs/http/ngx_http_v3_module.html) connections.

I use `enable_http3_global` to determine whether to enable [http3 directive](https://nginx.org/en/docs/http/ngx_http_v3_module.html) 
> Syntax:	http3 on | off;
Default:	
http3 on;
Context:	http, server